add docs with account upgrade instructions (#1223)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Accounts/cloud-upgrade-instructions.md

---

> title: "Upgrading your dbt Cloud Account"

Reviewer: ```suggestion
title: "Upgrade your dbt Cloud plan"
```

id: "cloud-upgrade-instructions"

> description: "Instructions for upgrading your dbt Cloud account after the trial ends."

Reviewer: ```suggestion
description: "Instructions for upgrading your dbt Cloud plan after the trial ends."
```

---
title: "How to upgrade a dbt Cloud account"
id: "cloud-upgrade-instructions"
description: "Instructions for upgrading a dbt Cloud account after the trial ends."
---

> dbt Cloud offers [several plans](https://www.getdbt.com/pricing/) with different features that meet your needs. This document is for dbt Cloud users and explains how to select a plan in order to continue using dbt Cloud. 

Reviewer 2: Maybe `admin` isn't the best word, but it may benefit us to use stronger language than `users` so it's apparent from the start that this isn't a task anyone can do (and lead into the `tip` that comes next).

```suggestion
dbt Cloud offers [several plans](https://www.getdbt.com/pricing/) with different features that meet your needs. This document is for dbt Cloud admins and explains how to select a plan to continue using dbt Cloud after the trial period ends.
```

Reviewer: thanks for this and good callout! i removed the 'afer the trial ends' because someone may need this even if they're not on trial. although that was the scenario that inspired sophia to write this doc, a user can change plans if their trial ends, their account is locked due to team plan cc failure, or they simply want to upgrade/downgrade their account.

:::tip Before you begin
- You **_must_** be part of the [Owner](/docs/collaborate/manage-access/self-service-permissions) user group to make billing changes. Users not included in this group will not see these options.
- All amounts shown in dbt Cloud are in U.S. Dollars (USD)   
:::

## Select a plan 

> When your [14 day trial](https://www.getdbt.com/signup/) ends or if your subscription payment is past due , you'll need to select a plan in order to continue using your account:

Reviewer 2: Is the behavior they will see exactly the same for the trial expiration and past due payment scenarios?

Reviewer: yes that's right! your account is locked

- Upon logging in, you should see an "Account locked" pop up message with instructions to unlock your account and update your payment details
- Click **Go to Billing** to go to the billing page
- Under **Billing**, you can review the available dbt Cloud [plans](https://www.getdbt.com/pricing/) and their features

To unlock your account and select a plan, review the following guidance per plan type:

### Developer plan

1. To select a Developer plan, choose the Developer tab under **Billing**. 
2. Click **Select Plan** on the right. 
3. Confirm your plan selection on the pop up message. 

> 4. This automatically unlocks your dbt Cloud account and you should now enjoy the benefits of the Developer plan. 🎉

Reviewer 2: ```suggestion
4. This automatically unlocks your dbt Cloud account, and you can now enjoy the benefits of the Developer plan. 🎉
```

<Lightbox src="/img/docs/dbt-cloud/downgrade-dev-flow.gif"/>

### Team plan

> 1. When your trial expires, your account's default plan enrollment will be a Team plan. 

Reviewer 2: If the account is locked, and they have to select a plan, what does it mean that the Team plan is `default`? It might  be good for readers for us to have this above the other instructions if this is the default behavior for all trials.

Reviewer: that's right! i initially only left it under the 'team plan' action because i didn't think users that have the intention of downgrading to dev would want to know this but will make that tweak

2. To unlock your account and continue using the Team plan, you need to enter your payment details. 
3. Go to **Payment Information** and click **Edit** on the right.
4. Enter your payment details and click **Save**.

> 5.  This automatically unlocks your dbt Cloud account and you should now enjoy the benefits of the Team plan. 🎉

Reviewer 2: ```suggestion
5.  This automatically unlocks your dbt Cloud account, and you can now enjoy the benefits of the Team plan. 🎉
```

<Lightbox src="/img/docs/dbt-cloud/trial-team-flow.gif"/>

### Enterprise plan

1. If you're interested in our Enterprise plan, select the Enterprise tab under **Billing**.
2. Click **Contact Sales** on the right. This opens a chat window for you to contact the dbt Cloud Support team, who will connect you to our Sales team.

> 3. Once you submit your request, our Sales team will contact you with more information. 

Reviewer 2: At this point, the product is locked, and they can't use it, so is there an estimated time we can provide for how long this process will take?

Reviewer: good question, i purposely left it out because i don't think we have an estimated time frame for this. i'd say probably a week but not sure if we want to hold ourselves to this. Unless using words like 'immediately contact you...' would that work?

Reviewer 2: It's good to me as is, I was just wondering if there was well defined, but it seems like it could be flexible.

    </div>
</details>
<details>
  <summary>Can I upgrade or downgrade my plan?</summary>
  <div>
    <div>Yes, you can upgrade or downgrade at any time. Account Owners can access their dedicated billing section via the account settings page.<br></br><br></br>
    
    If you’re not sure which plan is right for you, get in touch and we’ll be happy to help you find one that fits your needs.</div>
    </div>
</details>
<details>
  <summary>Can I pay by invoice?</summary>
    <div>
      <div>At present, dbt Cloud Team plan payments must be made via credit card, and by default they will be billed monthly based on the number of developer seats. <br></br><br></br>
      
      We don’t have any plans to do invoicing for Team plan accounts in the near future, but we do currently support invoices for companies on the dbt Cloud Enterprise plan. Feel free to <a href="https://www.getdbt.com/contact/">contact</a> us to build your Enterprise pricing plan.</div> 
    </div> 
</details>
<details>
  <summary>Why did I receive a <b>Failed payment</b> error email?</summary>
  <div>

>     <div>This means we were unable to charge the credit card you have on file or you have not provided an updated card for payment. If you're a current account owner with a card on file, contact your credit card issuer to inquire as to why your card was declined or update the credit card on your account.<br></br><br></br>

Reviewer 2: ```suggestion
    <div>This means we were unable to charge the credit card you have on file, or you have not provided an updated card for payment. If you're a current account owner with a card on file, contact your credit card issuer to inquire as to why your card was declined or update the credit card on your account.<br></br><br></br>
```
